lib/fog/ovirt/requests/compute/v4/create_vm.rb

```diff
@@ -3,26 +3,23 @@ module Compute
     class Ovirt
       class V4
         class Real
-          def check_for_option(opts, name)
-            opts[name.to_sym] || opts[(name + "_name").to_sym]
-          end
-
-          # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
-          def process_vm_opts(opts)
-            return unless check_for_option(opts, "template") && check_for_option(opts, "storagedomain")
+          def create_disk_attachment_from_disk(disk_to_attachment)
+            sparse = disk_to_attachment["sparse"].present? ? disk_to_attachment["sparse"] : true
```

Comment:
We can use `fetch` with default here.
```suggestion
            sparse = disk_to_attachment.fetch("sparse", true)
```